.github/workflows/probot-check-group.yml

```diff
@@ -0,0 +1,14 @@
+name: Probot
+
+on:
+  pull_request: {}
+
+jobs:
+  required-jobs:
+    runs-on: ubuntu-latest
+    # if this timeout triggers, then the job needs to be manually restarted through the GitHub interface
+    timeout-minutes: 60
+    steps:
+      - uses: carmocca/probot@v2
```

Comment:
What are the v1 & v2? Can we update all to v2?

Reply:
v1 has the cc-bot
v2 the check-group integration (WIP)

There's no need to upgrade if v1 works for cc-bot.